### DISCUSSION TITLE
Question 64: Typing into the table on the notes tab (LH-anthro)

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1071] [Q&A]
Description: We used to be able to type directly into the table on the notes tab, but now it’s grayed out and we can only enter data at the top. Can we change this back? 

In which module? Catalog

What time scale? CP : Last(ish) time he remembered editing was before the latest client.

**Comment by rondlg** [comment]
Response: This has been submitted to hornbill for Axiell to investigate -- ticket # [IN00221254](https://customer.hornbill.com/axiell/servicemanager/request/view/IN00221254/)

**Reply by magpiedin** [reply]
@lhancockFM -- From Axiell: It sounds like the EMu Catalog Note-text field (NotFisNotes0) can be made editable in the table again with the next client release.

If you/Chris also need other fields in that table ("Date" & "Type") to be editable, please let us know
